projects/components/src/summary-values/summary-values.component.ts

```diff
@@ -0,0 +1,56 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { TypedSimpleChanges } from '@hypertrace/common';
+import { SummaryValueDisplayStyle } from '../summary-value/summary-value.component';
+
+@Component({
+  selector: 'ht-summary-values',
+  styleUrls: ['./summary-values.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="ht-summary-values" data-sensitive-pii>
+      <ht-summary-value
+        [icon]="this.icon"
+        [value]="this.displayValue"
+        [label]="this.label"
+        [tooltip]="this.tooltip"
+        summaryValueDisplayStyle="${SummaryValueDisplayStyle.Text}"
+        class="summary-value"
+      ></ht-summary-value>
+      <ng-container *ngIf="this.additionalValues && this.additionalValues!.length > 0">
+        <div class="additional-values" [htTooltip]="detailsTemplate">
+          <span class="count">+{{ this.additionalValues!.length }}</span>
```

Comment:
So this would be `+3` for example, if given 4 values? Not really sure what this should look like, can you add a screenshot? here for example, I'd expect it to say `+3 more`

Reply:
The mocks says +3 only. Please check the dependent PR for screenshot

Reply:
I'll give you this one - the shading of the background conveys more, even if I prefer it to say it - but I think the single tooltip and keeping the default undefined behavior are good changes that don't affect your new use.

Reply:
I think the child summary value component should show '-' if the value is undefined. If we want to hide the entire component then that should be the responsibility of its parent component. Reason being, we may have use case where we are displaying a label and tooltip. In those case, displaying '-' instead of disappearing would be a good indication to the user.

Reply:
I don't think it's wrong for the summary value to do that, just that it's a change that's just as easy to do in the parent as the child. If you want to check all the usages to see if it makes sense though, I'm fine with it. Please combine the tooltip though - this should produce a single tooltip across all 3 elements.

Reply:
I have reverted the change for modifying summary value component. However, I am not sure about combining the tooltips. Mainly because, the tooltip on the first value can just represent the label. Like "IP Address" over the first value if we are not displaying the label itself. Combining the two also works, but we would need to define a display format for input tooltip and extra values. I think this is something we can tackle/iterate later on.

Reply:
would this work? I would prefer to merge this now and iterate on the other change. for now, it is matching the mocks.

Reply:
Combining them just makes the tooltip work like the inputs. Multiple values have been provided, so they should not be split into separate tooltips. If I'm going to take the effort to hover with my mouse, I want to see the whole list as one unit. In my mind, it's analogous to hiding part of a text string ellipses - we don't only show the hidden part when hovering, we show the whole string.

Reply:
done.